docs/data-sources/remote_network.md

```diff
@@ -35,3 +35,4 @@ data "twingate_remote_network" "foo" {
 ### Read-Only
 
 - `location` (String) The location of the Remote Network. Must be one of the following: AWS, AZURE, GOOGLE_CLOUD, ON_PREMISE, OTHER.
+- `type` (String) The type of the Remote Network. Must be one of the following: REGULAR, EXIT.
```

Comment:
@alexb-twingate was not sure how actually should we name new attribute **type** or **network_type**?